appdev/webapp.rst

```diff
@@ -13,17 +13,17 @@ In the end a .click package will be created, which can be uploaded to the OpenSt
 Download the following zip-file and unzip
 ::
 
-    https://github.com/Winael/ubuntu/blob/master/ubuntuphone/webapps/Reboot/alternate-webapp-generator.zip
+    wget https://raw.githubusercontent.com/Winael/ubuntu/master/ubuntuphone/webapps/Reboot/alternate-webapp-generator.zip
 
 Make the app-creator executable:
 ::
 
     chmod +x /path/to/alternate-webapp-generator.sh
```

Comment:
You could make this codeblock a little nicer if you also added the unzip command and joined them all into one block:
```
Download, extract and make the script executable::

    wget ...
    unzip...
    cd...
    chmod...

```

Reply:
Also, while technically correct to do it this way, I'd prefer that the two colons were at the end of the previous line.

```
Download the following zip-file and unzip::

    code

Make the app-creator executable::

    code
```